python/ray/train/tests/test_train_head.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import time
+
+import pytest
+import requests
+
+import ray
+from ray.train import RunConfig, ScalingConfig
+from ray.train.torch import TorchTrainer
+
+
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") == "1",
+    reason="This test is not supposed to work for minimal installation.",
+)
```

Comment:
Does this ever get run with this condition? I don't think any of the tests in `ray/train` should be?